daemons/ipa-kdb/ipa_kdb_kdcpolicy.c

```diff
@@ -63,8 +175,6 @@ ipa_kdcpolicy_check_tgs(krb5_context context, krb5_kdcpolicy_moddata moddata,
                         krb5_deltat *renew_lifetime_out)
 {
     *status = NULL;
-    *lifetime_out = 0;
-    *renew_lifetime_out = 0;
```

Comment:
why removing these ?

Reply:
I was thinking that this might be confusing since we are not changing lifteme and renewtime right now.

Reply:
In krb5 sometimes functions need to explicitly clear out passin arguments, so I would first check in the code if these are actually required, might be uninitialized in the KDC calling function...

Reply:
They are actually required.  I've posted a PR to change that [to krb5](https://github.com/krb5/krb5/pull/967), but it probably should stay for compat.